invenio_rdm_records/services/params/has_draft.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Facets parameter interpreter API."""
+
+from functools import partial
+
+from invenio_records_resources.services.records.params import ParamInterpreter
+
+
+class HasDraftParam(ParamInterpreter):
```

Comment:
Comment: Perhaps rename to Status since we call the parameter ``status``

Reply:
Quesiton: is it applying to both ``/api/records``  and ``/api/user/records``

Reply:
Question: Can you also check that the list of records is filtered to only what the user can actually edit

Reply:
I renamed it! The param is applied only in the user records resource. Also, I defined the `status` agg and post filter only for the user records resource.